### PR TITLE
Makes secglasses roundstart

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Replaces all instances of sunglasses & sechud in security lockers & job fills with secglasses.
Removes the research for secglasses since it is now pointless. To make them obtainable past roundstart, you can now craft secglasses by-hand using a sechud + glasses + some cable coil.

## Why / Balance
This has been discussed a few times before in the past, and the conclusion I've come to is that the tradeoff of sechud / glasses isn't really a very meaningful or interesting one. Making the person using the HUD unable to use flashes or flashbangs doesn't really do anything other then giving them slightly less options.
This is also pretty bad when you factor in that the sechud is pretty important QOL for a security officer. This ends up nerfing criminal records in a kind of unsavory way.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Security glasses have been moved from research to roundstart gear. All officers now start with them instead of sunglasses by default.
- add: You can now craft security glasses.

### CODE REVIEW

**Comment by Emisse** [comment]
i approve but yaml fail

**Comment by Hanzdegloker** [comment]
I just ask, sunglasses will be good flash protection and possible conversion protection and is obtainable outside of security. I want what happen to sechuds? what niche do they now fill if you get the full on upgrade off the rip when it comes to sec. Personally I would continue to use the sec hud out of looks but besides that, what role can the sec hud fill now?

**Comment by Flareguy** [comment]
> I just ask, sunglasses will be good flash protection and possible conversion protection and is obtainable outside of security. I want what happen to sechuds? what niche do they now fill if you get the full on upgrade off the rip when it comes to sec. Personally I would continue to use the sec hud out of looks but besides that, what role can the sec hud fill now?

If SS13 is to go off of, presumably none, outside of being the "base hud type" used in specific situations where that is warranted (like crafting.) It's not great, but in the current state of the sechud & sunglasses, there's really not much of a solution to this AFAIK.
There is at least the sec/med hud, which should help it feel a little less weird since you can't keep the flash protection if you're using that.

**Comment by K-Dynamic** [comment]
> I just ask, sunglasses will be good flash protection and possible conversion protection and is obtainable outside of security. I want what happen to sechuds? what niche do they now fill if you get the full on upgrade off the rip when it comes to sec. Personally I would continue to use the sec hud out of looks but besides that, what role can the sec hud fill now?

Maybe give them to sec auxiliary staff who you don't 100% trust with flash protection (i.e. lawyers) or you run out of secglasses

**Comment by IProduceWidgets** [comment]
I'm not 1000% sure, but I think your test fail is the construction graph.

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -163,6 +168,8 @@
   - type: FlashImmunity
   - type: EyeProtection
     protectionTime: 5
+  - type: Construction
+    graph: GlassesSecHUD
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:172]: needs to specify the node for the graph.
```suggestion
    graph: GlassesSecHUD
    node: glassesSec
```

there's your fail

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

**Comment by SpeltIncorrectyl** [comment]
I am very onboard with this, sechud cannot be used because the risk of being flashed is too great. Especially considering the time you really need to use sechud is the same time your enemies have flashes.

**Comment by notquitehadouken** [comment]
this is how it is like in /tg/ and i must say it is much better than locking them behind research

(also they look cool so hooray)

**Comment by Brandon-Huu** [comment]
How does this affect the secmed hud

**Comment by Flareguy** [comment]
> How does this affect the secmed hud

It doesn't touch anything relating to the hud itself. You can still print sechuds in the armory / get them from boxes.

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**